UnityProject/Assets/Scripts/Chat/Chat.Process.cs

```diff
@@ -87,19 +87,19 @@ private static (string, ChatModifier) ProcessMessage(ConnectedPlayer sentByPlaye
 		// Emote
 		if (message.StartsWith("*") || message.StartsWith("/me ",true,CultureInfo.CurrentCulture))
 		{
-			message = message.Substring(1);
+			message = message.Substring(message.StartsWith("/me ") ? 4 : 1);
```

Comment:
Maybe use `message.Replace("/me", "")` or `message.Remove(3)` instead, so the function is more obvious.